### PR TITLE
VACMS-4147: Add new fields to vha vocab.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #4147 

## Testing done
Visual / Behat

## Screenshots
image
image

## QA steps
- As an administrator, go to `admin/structure/taxonomy/manage/health_care_service_taxonomy/add` and visually verify new fieldset and fields appear as in above screenshot.

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml [hunk 1]
```diff
@@ -30,6 +34,23 @@ third_party_settings:
         open: false
       label: Governance
       region: hidden
+    group_vet_center:
+      children:
+        - field_service_type_of_care
```
Comment (kevwalsh) [inline config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml:39]: This is the wrong type of care field for this fieldset. should be `vet_center_type_of_care`.

File: config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml [hunk 2]
```diff
@@ -30,6 +34,23 @@ third_party_settings:
...
+        - field_vet_center_friendly_name
+        - field_vet_center_com_conditions
+        - field_vet_center_service_descrip
+      parent_name: ''
+      weight: 3
```
Comment (kevwalsh) [inline config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml:44]: Can you put this fieldgroup down below the other Description field? Basically, the Vet Center fields "override" the other fields, so best to have them last.

File: config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml [hunk 3]
```diff
@@ -30,6 +34,23 @@ third_party_settings:
...
+      format_type: details
+      region: content
+      format_settings:
+        description: ''
```
Comment (kevwalsh) [inline config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml:48]: Please add description (i forgot this in the AC)

"Add content to these fields if you want to "override" how the service is described on Vet Center pages."

**Review by kevwalsh — DISMISSED** [review]
@ethanteague A few little things here.

File: config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml:39]: Dangit - sorry about that, @kevwalsh

File: config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml (hunk 2 above)
Comment (ethanteague) [reply config/sync/core.entity_form_display.taxonomy_term.health_care_service_taxonomy.default.yml:44]: Sure, @kevwalsh

**Review by swirtSJW — APPROVED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**